feature-libs/cart/base/root/models/cart.model.ts

```diff
@@ -191,3 +192,28 @@ export enum CartValidationStatusCode {
   PRICING_ERROR = 'pricingError',
   UNRESOLVABLE_ISSUES = 'unresolvableIssues',
 }
+
+export interface AddToCartToastConfig {
+  timeout?: number;
+}
+
+export enum ADDED_TO_CART_FEEDBACK {
+  MODAL,
+  TOAST,
+  NONE,
+}
+export interface CartToastItem {
+  productName: string;
+  quantity: number;
+  unitPrice: string;
+  baseClass: string;
+  images?: Images;
+  timeoutRef?: ReturnType<typeof setTimeout>;
+  scrollUnlistener?: () => void;
+}
+
+export enum CART_TOAST_STATE {
```

Comment:
CART_TOAST_STATE doesn't seem like a model that has a use outside of the AddToCartToastComponent.